contracts/sources/lending_market.move

```diff
@@ -819,6 +843,22 @@ module suilend::lending_market {
         reserve::update_reserve_config<P>(reserve, config);
     }
 
+    public fun set_emode_for_pair<P, T>(
```

Comment:
i think it makes sense to get the type argument for the pair reserve as well, just as a sanity check.

Reply:
Added both:

```
public fun set_emode_for_pair<P, Deposit, Borrow>(
        _: &LendingMarketOwnerCap<P>, 
        lending_market: &mut LendingMarket<P>, 
        deposit_reserve_array_index: u64,
        borrow_reserve_array_index: u64,
        open_ltv_pct: u8,
        close_ltv_pct: u8,
    ) {
        assert!(lending_market.version == CURRENT_VERSION, EIncorrectVersion);

        let borrow_reserve = vector::borrow_mut(&mut lending_market.reserves, borrow_reserve_array_index);
        assert!(reserve::coin_type(borrow_reserve) == type_name::get<Borrow>(), EWrongType);
        
        let deposit_reserve = vector::borrow_mut(&mut lending_market.reserves, deposit_reserve_array_index);
        assert!(reserve::coin_type(deposit_reserve) == type_name::get<Deposit>(), EWrongType);
        

        reserve::set_emode_for_pair<P>(deposit_reserve, borrow_reserve_array_index, open_ltv_pct, close_ltv_pct);
    }
```